### PR TITLE
chore: set dummy NEW_RELIC_LICENSE_KEY

### PR DESCRIPTION
**Pull request by keroro520**
A dummy `NEW_RELIC_LICENSE_KEY` is set in this PR to simplify the way we use NewRelic:

- 33112c5c0ff2c80daff4613db68802c58daf4416 feat: set dummy `NEW_RELIC_LICENSE_KEY` '0000000000000000' by default

  By using this setting, we don't have to check `envConfig.newRelicLicenseKey` every time we want to use NewRelic.

  PS: I have no idea how the dummy key works. It worked when I applied the suggestion in [this post](https://discuss.newrelic.com/t/dummy-license-key-for-testing-ins)

### CODE REVIEW

**Comment by Flouse** [comment]
> * PS: I have no idea how the dummy key works. It worked when I applied the suggestion in [this post](https://discuss.newrelic.com/t/dummy-license-key-for-testing-ins)

-> Oops! That page doesn’t exist or is private.

**Review by Flouse — APPROVED** [review]

File: packages/api-server/src/base/env-config.ts [hunk 1]
```diff
@@ -6,7 +6,6 @@ dotenv.config({ path: "./.env" });
 export const envConfig = {
   databaseUrl: getRequired("DATABASE_URL"),
   godwokenJsonRpc: getRequired("GODWOKEN_JSON_RPC"),
-  newRelicLicenseKey: getOptional("NEW_RELIC_LICENSE_KEY"),
```
Comment (RetricSu) [inline packages/api-server/src/base/env-config.ts:9]: let's keep this variable in envConfig? since `.env` in `README.md` has such config?

**Review by RetricSu — APPROVED** [review]

**Comment by keroro520** [comment]
Included in https://github.com/nervosnetwork/godwoken-web3/pull/444

File: packages/api-server/src/base/env-config.ts (hunk 1 above)
Comment (keroro520) [reply packages/api-server/src/base/env-config.ts:9]: Updated in another PR:
https://github.com/nervosnetwork/godwoken-web3/pull/444/files#diff-e5de419b6edadcf4d59c1cf7d75c9d50d2a567039753911c63d7eec4aaf0c75bR9

```diff
-  newRelicLicenseKey: getOptional("NEW_RELIC_LICENSE_KEY"),
+  _newRelicLicenseKey: getOptional("NEW_RELIC_LICENSE_KEY"),
```

**Pull request closed without merging**